lib/wds-components/app/templates/components/wds-page-header.hbs

```diff
@@ -0,0 +1,36 @@
+{{#if model.wordmark.image-url}}
+	<div class="wds-page-header__wordmark">
+		<a accesskey="z" href="{{model.wordmark.href}}">
+			<img src="{{model.wordmark.image-url}}" alt="{{model.wordmark.sitename}}" width="250" height="65">
+		</a>
+	</div>
+{{/if}}
+
+<div class="wds-page-header__top-container">
+	<div class="wds-page-header__wordmark-text">
+		<a href="{{model.wordmark.href}}">{{model.wordmark.sitename}}</a>
```

Comment:
semantically this section has nothing to do with wordmark, it's just sitename, so I think that class name should be something link wds-page-heade__sitename. And maybe keys in model should be changed also - to consider.